src/views/poi_bloc/Images.jsx

```diff
@@ -0,0 +1,55 @@
+/* global _ */
+import React from 'react';
+import classnames from 'classnames';
+
+function createImage(param, classes) {
+  return <a
+    target='_blank'
+    rel='noopener noreferrer'
+    href={ param.href }
+    className={ classes }
+    style={{ backgroundImage: param.url }}
+  />;
+}
+
+function createImages(images) {
+  if (images.length < 2) {
+    return;
+  } else if (images.length === 2) {
+    return <div className='poi_panel__pictures_block poi_panel__pictures_second_block'>
+      { createImage(images[1], 'poi_panel__pictures_block_full') }
+    </div>;
+  }
+  return <div
+    className={ classnames(
+      'poi_panel__pictures_block',
+      'poi_panel__pictures_second_block',
+      'poi_panel__pictures_block-multiple') }>
+    { createImage(images[1], 'poi_panel__pictures_block_short') }
+    { createImage(images[2], 'poi_panel__pictures_block_short') }
+  </div>;
+}
+
+const ImagesBlock = ({
+  block,
+  poi,
+  asString,
+}) => {
+  const images = block.images.filter(img => img.url !== poi.topImageUrl);
+  if (asString) {
+    return '';
+  } else if (!images || images.length === 0) {
```

Comment:
These two conditions can be grouped with the previous test.

Reply:
I voluntarily didn't group them to keep both code logics separated.

Reply:
Actually, after checking it, I would just remove the `asString` prop completely, because the `images_panel` doesn't implement `toString` so this component doesn't need to implementthat.

And for the other test, it's better to return `null`, because with the empty string React generates stuff in the DOM (historically an empty `<span>`, now comments).